src/ansys/geometry/core/tools/repair_tools.py

```diff
@@ -529,6 +532,20 @@ def find_and_fix_short_edges(
 
         # Update existing design
         parent_design = get_design_from_body(bodies[0])
+
+        if not pyansys_geometry.USE_TRACKER_TO_UPDATE_DESIGN:
+            parent_design._update_design_inplace()
+        else:
+            parent_design._update_from_tracker(response["complete_command_response"])
+
+        message = RepairToolMessage(
+            success=response["success"],
+            found=response["found"],
+            repaired=response["repaired"],
+            created_bodies=[],
+            modified_bodies=[],
+        )
+        return message
         parent_design._update_design_inplace()
 
```

Comment:
There seems to be some unreachable code here @umutsoysalansys - I think this part of the code needs some attention. Could you review it?

Reply:
cleaned, the merge artifacts.